demisto_sdk/commands/common/hook_validations/correlation_rule.py

```diff
@@ -84,3 +85,16 @@ def is_files_naming_correct(self):
                 self._is_valid = False
                 return False
         return True
+
+    @error_codes("CR102")
+    def validate_execution_mode_search_window(self):
+        """
+        Validates 'search_window' existence and non-emptiness for 'execution_mode' = 'SCHEDULED'.
+        """
+        if (not 'search_window' in self.current_file or
```

Comment:
```suggestion
        if (('search_window' not in self.current_file) or
```
unless it's not what you meant

Reply:
it is still a draft PR